src/screens/augmentedReality/ARShowScreen.js

```diff
@@ -122,9 +122,20 @@ const objectParser = async ({ item, setObject, setIsLoading, onPress }) => {
 
   item?.payload?.localUris?.forEach((item) => {
     if (item.type === 'texture') {
-      parsedObject[item.type].push({ uri: item.uri });
+      parsedObject[item?.type].push({ uri: item?.uri });
     } else {
-      parsedObject[item.type] = item.uri;
+      parsedObject[item?.type] = {
```

Comment:
why did you add the `?` here for the object keys `parsedObject[item?.type]`?
think about not having an `item`, this will become undefined. so you have `parsedObject[undefined]` which makes no sense. if you want to be sure, that `item` exists in order to access `type` you need to check that explicitly before pushing or assigning.

Reply:
Yes, you're right. 
https://github.com/ikuseiGmbH/smart-village-app-app/pull/634/commits/689faf5322a8c44097e267c3ca81f3b154d94ccd